spek-kotlin-compiler-plugin-native/src/main/kotlin/org/spekframework/spek2/kotlin/SpekExtension.kt

```diff
@@ -156,7 +153,9 @@ private class SpekCollector(
     }
 
     private val IrClass.isSpek: Boolean
-        get() = superTypes.any { it.classifierOrFail.descriptor.fqNameSafe.asString() == spekClassName }
+        get() = superTypes.any {
+            it.getClass()?.fqNameForIrSerialization?.asString() == spekClassName
```

Comment:
not related to this PR, removes the deprecation warning for `it.classifierOrFail.descriptor`.